client/jetpack-connect/auth-form-header.jsx

```diff
@@ -107,7 +107,7 @@ export class AuthFormHeader extends Component {
 			switch ( currentState ) {
 				case 'logged-out':
 					return translate(
-						'Your account will enable you to start using the features and benefits offered by Jetpack & WooCommerce Services.'
+						'Your Jetpack account will enable you to start using the benefits offered by Jetpack & WooCommerce Services.'
```

Comment:
:disappointed: 19 existing translations will be lost with this change.

Reply:
I was originally changing these strings to match the mocks, but I think they are similar enough that we should stick with the existing.